ui-lib/src/androidTest/java/co/electriccoin/zcash/ui/screen/seed/view/SeedViewSecuredScreenTest.kt

```diff
@@ -0,0 +1,48 @@
+package co.electriccoin.zcash.ui.screen.seed.view
+
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.test.junit4.ComposeContentTestRule
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.test.filters.MediumTest
+import cash.z.ecc.sdk.fixture.PersistableWalletFixture
+import co.electriccoin.zcash.test.UiTestPrerequisites
+import co.electriccoin.zcash.ui.common.LocalScreenSecurity
+import co.electriccoin.zcash.ui.common.ScreenSecurity
+import co.electriccoin.zcash.ui.design.theme.ZcashTheme
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import org.junit.Rule
+import org.junit.Test
+import kotlin.test.assertEquals
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class SeedViewSecuredScreenTest : UiTestPrerequisites() {
```

Comment:
Great first test case!

The decision to make this a separate class (instead of just additional methods on the existing class) was a good decision, as it keeps the existing tests simple.